tests/test_cli_module_test.py

```diff
@@ -7,17 +7,25 @@
 
 
 class TestTestCli(unittest.TestCase):
-    @mock.patch("taskcat._cli_modules.test.Config", autospec=True)
-    @mock.patch("taskcat._cli_modules.test.LambdaBuild", autospec=True)
-    @mock.patch("taskcat._cli_modules.test.ReportBuilder", autospec=True)
-    def test_test_run(self, mock_report_builder, mock_lambda_build, mock_config):
+    @mock.patch("taskcat._cli_modules.test.TestManager", autospec=True)
+    def test_test_run(self, mock_manager):
         base_path = "./" if os.getcwd().endswith("/tests") else "./tests/"
         base_path = Path(base_path + "data/nested-fail").resolve()
+        input_path = base_path / ".taskcat.yml"
 
-        Test.run(project_root=base_path, input_file=base_path / ".taskcat.yml")
-        mock_report_builder.assert_called()
-        mock_lambda_build.assert_called()
-        mock_config.create.assert_called()
+        Test.run(project_root=base_path, input_file=input_path)
+
+        # Make sure we created test_manager using from_file
+        mock_manager.from_file.assert_called_once_with(
+            base_path, input_path, "ALL", False
+        )
+
+        # Mock manager is the class, here we get an instance of it
+        test_manager = mock_manager.from_file.return_value
+
+        test_manager.start.assert_called_once_with("ALL", "ALL", False, False)
+        test_manager.report.assert_called_once_with("./taskcat_outputs")
+        test_manager.end.assert_called_once_with(False, False, False)
```

Comment:
Updated the existing test for the `run` subcommand now that it uses the `TestManager` class.